### DISCUSSION TITLE
macos runtime error

### DISCUSSION THREAD

**Discussion by materia-nigra** [madMAx43v3r/chia-plotter#356] [Q&A]
I got this message while trying to make the first plot. I have no clue what's going on.

> libc++abi.dylib: terminating with uncaught exception of type std::runtime_error: fopen() failed
> zsh: abort

**Comment by Bingo821227** [comment]
same question

**Comment by HeyManGonzo** [comment]
I got this on a 2018 macbook pro i7/16RAM: 
libc++abi: terminating with uncaught exception of type std::runtime_error: thread failed with: fwrite() failed

**Reply by SebMoore** [reply]
> I got this on a 2018 macbook pro i7/16RAM:
> libc++abi: terminating with uncaught exception of type std::runtime_error: thread failed with: fwrite() failed

Ok your problem appears to be that you have run out of disk space. Please ensure that you have at least around 256gb if not using ramdisk, and 220 if using ramdisk.
See: #53, https://github.com/madMAx43v3r/chia-plotter/issues/109#issuecomment-859037767, #129, #333

**Reply by HeyManGonzo** [reply]
Thanks! That was totally my mistake. Temp 1 disk did indeed not have space enough!

**Reply by SebMoore** [reply]
> Thanks! That was totally my mistake. Temp 1 disk did indeed not have space enough!

No worries! Glad to help.

**Comment by SebMoore** [comment]
Please, @fritzpfritz @Bingo821227 @HeyManGonzo  post the exact commands you used to run the plotter as these usually reveal the issue.

**Reply by HeyManGonzo** [reply]
I used the following command: ./build/chia_plot -n 1 -r 10 -u 128 -t /Volumes/T5Mac/temp1/ -2 /Volumes/Freecom1TB/temp2/ -d /Volumes/Freecom1TB/plots/ -p xxx -f xxx

The eror I got was during [P1] Table 6 as table 5 was already completed 917 seconds.

**Reply by HeyManGonzo** [reply]
The eror I got was during [P1] Table 6 as table 5 was already completed 917 seconds.

**Comment by moechenche** [comment]
I get this error when my Raid0 fails it usually happens in Phase 1 but sometimes not till Phase 2
